# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/tests/editing/crashtests/remove-editing-host-on-DOMNodeInserted-at-indent.html
# Commit: 69591833b1c2
# Full Hash: 69591833b1c2db9884926bdeadbe3183d17fb63f
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2022-12-20 09:39:56
# Description:
#   Bug 1805554 - Make `HTMLEditor::MaybeSplitAncestorsForInsertWithTransaction` return `NS_ERROR_EDITOR_UNEXPECTED_DOM_TREE` if it got lost the editing host r=m_kato
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D164771
# ==============================================================================

diff -r 857b1d139d11 -r 69591833b1c2 testing/web-platform/tests/editing/crashtests/remove-editing-host-on-DOMNodeInserted-at-indent.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/testing/web-platform/tests/editing/crashtests/remove-editing-host-on-DOMNodeInserted-at-indent.html	Tue Dec 20 03:13:57 2022 +0000
@@ -0,0 +1,24 @@
+<!doctype html>
+<html>
+<head>
+<meta charset="utf-8">
+<script>
+document.addEventListener("DOMContentLoaded", () => {
+  const editingHost = document.querySelector("h6[contenteditable]");
+  document.querySelector("wbr").addEventListener(
+    "DOMNodeInserted",
+    () => editingHost.remove()
+  );
+  window.find("A");
+  document.execCommand("indent");
+})
+</script>
+</head>
+<body>
+<h6 contenteditable>
+<time>
+<wbr>
+<main></main>
+A
+</time></h6></body>
+</html>
